programs/clearing_house/src/state/events.rs

```diff
@@ -131,6 +131,7 @@ pub enum OrderAction {
 #[derive(Clone, Copy, BorshSerialize, BorshDeserialize, PartialEq, Eq)]
 pub enum OrderActionExplanation {
     None,
+    InsufficientFreeCollateral,
     BreachedMarginRequirement,
```

Comment:
doesnt look like `BreachedMarginRequirement` gets used, should we remove?

Reply:
sure